recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 7ef4694e1345913182126b219aaa4a0047e191af414256da6772cf249571b961
 
 build:
-  number: 1
+  number: 2
```

Comment:
There is no need, but it does not hurt to leave it there, because there is no build 1 for `OS X`. This will create builds `2` for both Linux and Windows that are identical to build 1 though.